pages/en/billing.mdx

```diff
@@ -0,0 +1,76 @@
+
+There are 4 states your invoice can be in:
+
+1. Created - your invoice has just been created and not been paid yet
+2. Paid - your invoice has been successfully paid
+3. Unpaid - there is not enough GRT in your balance on the billing contract
+4. Error - there is an error processing the payment
+
+**See the diagram below for more information:**
+
+![Billing Flow](/img/billing-flow.png)
+
+### Adding GRT to Account
+
+Before making queries on The Graph Network, you will need to add GRT to your account. To add GRT to your account, take the following steps:
+
+1. Purchase GRT and ETH on an exchange of your choice
+2. Send the GRT and ETH to your wallet
+3. Bridge GRT to Polygon using the Subgraph Studio UI
+
+   a) You will receive 0.001 Matic within a few minutes of sending GRT to the Polygon bridge. You can monitor the transaction by inputting your wallet address into the search bar on [Polygonscan](https://polygonscan.com/).
+
+4. Add bridged GRT to the billing contract on Polygon. The billing contract address is: [0x10829DB618E6F520Fa3A01c75bC6dDf8722fA9fE](https://polygonscan.com/address/0x10829DB618E6F520Fa3A01c75bC6dDf8722fA9fE).
+
+   a) In order to complete step #4, you'll need to switch your network in your wallet to Polygon. You can add Polygon's network by connecting your wallet and clicking on "Choose Polygon (Matic) Mainnet" [here](https://chainlist.org/). Once you've added the network, switch it over in your wallet by navigating to the network pill on the top right-hand side corner. In Metamask, the network is called **Polygon Mainnnet**.
+
+If you used your API keys over the course of any given week, you will receive an invoice based on the query fees you have generated during this period. This invoice will be paid using GRT available in your balance. Query volume is evaluated by the API keys you own. Your balance will be updated after fees are withdrawn.
+
+### Understanding Invoices in Subgraph Studio
+
+When using The Graph Network, dapps and users pay for query fees in GRT, the protocol's native work-utility token. Maintaining a positive balance for an account is the user's or dapp's responsibility.
+
+Here are ways to be notified about and/or check a query fee balance:
+
+- You can set up a billing alert that will be sent to an email address [here](https://thegraph.com/studio/settings/).
+
+- You can query the billing subgraph API directly [here](https://thegraph.com/hosted-service/subgraph/graphprotocol/billing?query=Nena%20account)
+
+Also, you can learn how to (re)fill your query fee balance by watching [this video](https://www.loom.com/share/a81de6ef11d64c62872ea210c58c6af5).
+
+> IMPORTANT: When an account reaches a 0 GRT balance and the billing invoice is not paid in a timely manner, Indexers will stop serving queries for all API Keys associated with the account. Indexers will automatically start serving queries again once the billing balance is paid in full and GRT is added to the account for future query fees.
```

Comment:
Not sure if we should mention that we are giving users a 2 weeks "grace period". After 2 weeks of unpaid invoices - we'll stop your queries. If they enabled email notifications, we'll send a few emails prior to that cc @azf20 @schmidsi

Reply:
I think it's ok to be slightly vague here, but we may want to add more specifics if users are confused about `timely manner`

Reply:
Replaced `timely manner` with 2 weeks - not sure how to be more specific without giving them the number. Just feels out of place to write "in a few weeks." 

I don't think it's a bad idea to include the 2 weeks since it's only mentioned this one time in the docs - unless we think this will encourage users to gamify the protocol?

Reply:
Talked to Kyle and he suggested saying "if there are 2 unpaid invoices" . so it's clear and we don't talk about the time, win-win cc @MichaelMacaulay

Reply:
Good call - thank you!